test/clj/rems/email/test_template.clj

```diff
@@ -187,8 +189,18 @@
     (is (= #{"assistant" "handler"} (email-recipients mails)))
     (is (= {:to-user "assistant"
             :subject "(2001/3, \"Application title\") Application has been commented"
-            :body "Dear Amber Assistant,\n\nremarker has commented on the application 2001/3, \"Application title\", submitted by Alice Applicant.\n\nYou can review the application at http://example.com/application/7"}
-           (email-to "assistant" mails)))))
+            :body "Dear Amber Assistant,\n\nRandom Remarker has commented on the application 2001/3, \"Application title\", submitted by Alice Applicant.\n\nYou can review the application at http://example.com/application/7"}
+           (email-to "assistant" mails))))
+  (let [mails (emails base-events {:application/id 7
+                                   :event/type :application.event/remarked
+                                   :event/actor "remarker"
+                                   :application/public true
+                                   :application/comment "remark!"})]
+    (is (= #{"assistant" "handler" "applicant"} (email-recipients mails)))
+    (is (= {:to-user "applicant"
```

Comment:
Do we have some test that checks member mails too?

Reply:
the bodies are known to be the same so the tests check only one

Reply:
Oh right, this application doesn't have any members.